UPGRADE-6.1.md

```diff
@@ -0,0 +1,8 @@
+UPGRADE FROM 6.0 to 6.1
+=======================
+
+Security
+--------
+
+ * Deprecate `RememberMeHandler` not implementing `getUserIdentifierForCookie()`; `AbstractRememberMeHandler` has a default implementation
+ * Deprecate `RememberMeHandler` not implementing `getRememberMeDetails()`; `AbstractRememberMeHandler` has a default implementation
```

Comment:
Yay, the first 6.1 changes! Can you please also create an `UPGRADE-7.0.md` document, documenting the results after removing the BC layer in the future? (e.g. "Added `getUserIdentifierForCookie()` to `RememberMeHandler`").

Reply:
I've added the file. Mentioned the methods are added to `RememberMeHandlerInterface`.